### PR TITLE
[MBL-17612][Parent] Create account flow

### PR DESCRIPTION
**Pull request by andrasmaczak**
Test plan: 
- On landing page select QR code
- Select I don't have a Canvas account
- Follow create account flow
- Try logging in with the new parent user

refs: MBL-17612
affects: Parent
release note: none

## Checklist

- [x] Tested in dark mode
- [x] Tested in light mode

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Review by tamaskozmer — CHANGES_REQUESTED** [review]
QA findings:

- [ ] Font seems wrong on the 'Open Canvas Student' app screen, I know this is an old issue. (This might get fixed in MBL-18086)
- [x] Red color in textfield validation is too dark in dark mode. It looks the same in light mode, I guess we are using a default color and not using our own error color.
- [x] Bottom sheet text is too small and have the wrong font.
- [x] Bottom sheet items are only clickable when the actual text is clicked. The whole row should be clickable and have touch feedback. Also touch feedback is strange it can only be seen when long tapped on the text.
- [x] Soft keyboard covers the screen when a textfield is in focus instead of pushing it up.
- [x] Bottom sheet should be dismissed when navigating to the next screen.

File: apps/parent/src/main/java/com/instructure/parentapp/features/login/createaccount/CreateAccountScreen.kt [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+                            )
+                        }
+                        Spacer(modifier = Modifier.height(30.dp))
+                        Row(
+                            verticalAlignment = Alignment.CenterVertically,
+                            horizontalArrangement = Arrangement.Center
+                        ) {
+                            Text(
+                                text = stringResource(R.string.createAccAlreadyHaveAccount),
+                                color = colorResource(R.color.ash)
```
Comment (tamaskozmer) [inline apps/parent/src/main/java/com/instructure/parentapp/features/login/createaccount/CreateAccountScreen.kt:144]: This should also use `textDark` and `textInfo` for the link.

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kristofnemere — APPROVED** [review]
QA+1

File: apps/parent/src/androidTest/java/com/instructure/parentapp/ui/pages/CreateAccountPage.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    fun fillInvalidData() {
+        composeTestRule.onNodeWithTag("emailInput").performTextInput("email")
+        composeTestRule.onNodeWithTag("passwordInput").performTextInput("pwd")
+    }
+
+    fun clickCreateAccountButton() {
+        composeTestRule.onNodeWithText("Create Account").performClick()
+    }
+
+    fun isRecipientsLoading(): Boolean {
```
[the remark this answers is not in the document]
Comment (andrasmaczak) [reply apps/parent/src/androidTest/java/com/instructure/parentapp/ui/pages/CreateAccountPage.kt:52]: Busted

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Review by kristofnemere — APPROVED** [review]
QA+1

**Pull request merged**